frontend/src/toolbar/flags/FeatureFlags.tsx

```diff
@@ -3,131 +3,103 @@ import './featureFlags.scss'
 import { toolbarLogic } from '~/toolbar/toolbarLogic'
 import { urls } from 'scenes/urls'
 import { IconOpenInNew } from 'lib/components/icons'
 
 export function FeatureFlags(): JSX.Element {
-    const { showLocalFeatureFlagWarning, searchTerm, filteredFlags } = useValues(featureFlagsLogic)
-    const { setOverriddenUserFlag, deleteOverriddenUserFlag, setShowLocalFeatureFlagWarning, setSearchTerm } =
-        useActions(featureFlagsLogic)
-    const { apiURL, posthog } = useValues(toolbarLogic)
+    const { searchTerm, filteredFlags } = useValues(featureFlagsLogic)
+    const { setOverriddenUserFlag, deleteOverriddenUserFlag, setSearchTerm } = useActions(featureFlagsLogic)
+    const { apiURL } = useValues(toolbarLogic)
 
     return (
         <div className="toolbar-block">
-            {showLocalFeatureFlagWarning ? (
-                <div className="local-feature-flag-override-warning">
-                    <Typography.Text>
-                        <Typography.Text type="warning" strong>
-                            Warning:
-                        </Typography.Text>{' '}
-                        It looks like you've previously used the developer console to set local feature flags. To use
-                        feature flags in the toolbar, please clear them below.
-                    </Typography.Text>
-                    <div>
-                        <Button
-                            type="primary"
-                            onClick={() => {
-                                posthog?.feature_flags.override(false)
-                                setShowLocalFeatureFlagWarning(false)
-                            }}
-                        >
-                            Clear locally set feature flags
-                        </Button>
-                    </div>
-                </div>
-            ) : (
-                <>
-                    <Input.Search
-                        allowClear
-                        autoFocus
-                        placeholder="Search"
-                        value={searchTerm}
-                        className={'feature-flag-row'}
-                        onChange={(e) => setSearchTerm(e.target.value)}
-                    />
-                    <List
-                        dataSource={filteredFlags}
-                        renderItem={({
-                            feature_flag,
-                            value_for_user_without_override,
-                            override,
-                            hasVariants,
-                            currentValue,
-                        }) => {
-                            return (
-                                <div className={'feature-flag-row'}>
+            <div className="local-feature-flag-override-note">
+                <Typography.Text>
+                    Note, overriding feature flags below will only impact the experience within this browser.
```

Comment:
nit... Most people's reading age is surprisingly low

Is it clearer to say "Note, overriding feature flags below will only affect this browser."